doc/conf.py

```diff
@@ -181,7 +181,7 @@
     }
 )
 intersphinx_mapping["mne-gui-addons"] = ("https://mne.tools/mne-gui-addons", None)
-intersphinx_timeout = 5
```

Comment:
for some reason is taking longer in the CI, just increase the timeout seems to be solving locally